app.yml

```diff
@@ -0,0 +1,4 @@
+default_events:
+- issues
+default_permissions:
+  issues: write
```

Comment:
What would you think about adding a template app.yml like [this one](https://github.com/probot/create-probot-app/blob/master/templates/basic-js/app.yml)? That way all the options are there for them if they want to look through them to create an app that has other permissions.

If we don't do the template, we should also add `metadata: read` to this, since you can sometimes get errors if you don't add it. It's also the default in our UI to enable that permission.